Source/Core/Common/Logging/LogTypes.h

```diff
@@ -0,0 +1,60 @@
+// Copyright 2009 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+// no header guard, we WANT to include this multiple times (with different implementation of the X
```

Comment:
Do we want define guards instead?
```c
#ifndef TYPE
#error TYPE not defined
#endif
```

Reply:
Good idea. Might be useful to attach a bit of documentation too (e.g. "TYPE must be defined as a macro taking a log type identifier, short name and full name").